lnotify: Fix undefined ignore_windows_list

mkaito opened the pull request:
If the first `if` returns `False`, the second `if` will explode because
`ignore_windows_list` will never be defined.

oakkitten commented on python/lnotify.py:
```diff
@@ -101,9 +101,9 @@ def handle_msg(data, pbuffer, date, tags, displayed, highlight, prefix, message)
-    if ignore_windows_list in window_name:
-        x_focus = True
-        return weechat.WEECHAT_RC_OK
+        if ignore_windows_list in window_name:
```
shouldn't it be the other way around, `window_name in ignore_window_list` anyway

mkaito replied:
Uh, probably, yes. I just fixed the indentation because it causes an exception.

mkaito commented:
Uh, probably, yes. I just fixed the indentation because it causes an exception.

El 17 de octubre de 2018 17:09:49 GMT+01:00, oakkitten <notifications@github.com> escribió:
>oakkitten commented on this pull request.
>
>
>
>> @@ -101,9 +101,9 @@ def handle_msg(data, pbuffer, date, tags,
>displayed, highlight, prefix, message)
>         cmd_name=("ps -ho comm -p %s"%(window_pid)).split()
>         window_name = subprocess.check_output(cmd_name)
>         ignore_windows_list = ["tilda", "gnome-terminal", "xterm"]
>-    if ignore_windows_list in window_name:
>-        x_focus = True
>-        return weechat.WEECHAT_RC_OK
>+        if ignore_windows_list in window_name:
>
>shouldn't it be the other way around, `window_name in
>ignore_window_list` anyway
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/weechat/scripts/pull/296#pullrequestreview-165715938

-- 
Enviado desde mi dispositivo Android con K-9 Mail. Por favor, disculpa mi brevedad.

mkaito commented:
/bump

flashcode commented:
Hi,
Could you please rebase all your commits in a single one, and use the commit message described in the Contributing file?
Thanks.

mkaito commented:
Done.

flashcode commented:
Merged, thanks.
